simpledotorg/simple-server
# New "Progress tab" home page

**Story card:** [sc-6923](https://app.shortcut.com/simpledotorg/story/6923/create-new-progress-home-page?vc_group_by=day&ct_workflow=all&cf_workflow=500000031)

## Because

Building the new "Progress" tab home page (without the actual data). Releasing behind the `:new_progress_tag` Flipper flag.

## This addresses
![image](https://user-images.githubusercontent.com/16785131/153017090-f6ae38ad-f73d-4d28-813c-e5bd8a5529ab.png)

* New "Reports" card
* Recently restyled "Drug stock report" card
* New "Achievements" card

## Test instructions

* First, go to the Dashboard report of any facility
* Click the "Progress reports" tab
* See the view's mobile version. The page should display the current progress tab designs/data.
* Turn on the `:new_progress_tab` feature flag
* Refresh the progress report page and check the new designs. The buttons shouldn't work and the data is false. This is just a static version of the new home page.
